File: server/Models/Vault.cs
namespace keeper.Models;

public class Vault : RepoItem<int>
{
    public string Name { get; set; }
    public string Description { get; set; }
    public string Img { get; set; }
    public string CreatorId { get; set; }
    public Account Creator { get; set; }
    public bool IsPrivate { get; set; }
}

File: client/src/models/Vault.js
export class Vault {
    constructor(data) {
        this.id = data.id
        this.name = data.name
        this.description = data.description
        this.img = data.img
        this.creatorId = data.creatorId
        this.creator = data.creator
        this.isPrivate = data.isPrivate
    }
}